src/Business Foundation/App/NoSeries/src/Setup/NoSeries.Page.al

```diff
@@ -292,4 +324,38 @@ page 456 "No. Series"
         NoSeriesSingle := Implementation;
         AllowGaps := NoSeriesSingle.MayProduceGaps();
     end;
+
+    local procedure ShowNoSeriesWithWarningsOnly(ShowWarningsOnly: Boolean)
+    var
+        NoSeriesLine: Record "No. Series Line";
+        NoSeries: Codeunit "No. Series";
+        LastNoUsedForLine: Code[20];
+    begin
+        ShowNoSeriesWithWarnings := ShowWarningsOnly;
```

Comment:
What do we need the global variable `ShowNoSeriesWithWarnings` for?

Reply:
Was needed in a previous design but not anymore, will remove